content/pages/events.rst

```diff
@@ -141,14 +141,15 @@ Friday Delft Bicycle and Walk Tour
 ==================================
 
 | **Location:** Delft City Center
-| **Date:** Friday 19, October 2023, 15:00-18:00h
+| **Date:** Friday 20, October 2023, 14:00-15:30h
```

Comment:
Correct Date is Friday 20

Reply:
If this even is only 1.5 hour long, then I would like to have it start at 15:00 and end at 16:30. This gives us one more hour in the conference and some more flexibility in the conference talk schedule.

Is that possible?

Reply:
I would say yes. I don't know in October, what is the daylight? May be convenient to shift to 14.30 - 16 ? To visit the city centre with sun still "high", that's my only concern

Reply:
Nice, so let's start at 15